packages/dds/tree/src/feature-libraries/chunked-forest/chunkTree.ts

```diff
@@ -197,6 +197,13 @@ export function chunkFieldSingle(
 	policy: ChunkCompressor,
 ): TreeChunk {
 	const chunks = chunkField(cursor, policy);
+	return combineChunks(chunks);
+}
+
+/**
+ * Combine multiple TreeChunks into a single TreeChunk.
```

Comment:
Since this can take in 0 chunks, I'm not sure this phrasing is the best.

Also, the ownership of the chunks involved is not very clear.

I think this has the semantics that it takes ownership of the input, and returns owning reference.

Maybe this would be better:


```suggestion
 * Create a single TreeChunk from an array of TreeChunks.
 *  @remarks
 * This takes ownership of the provided TreeChunk references, and returns an owned referenced.
```